--- a/configure
+++ b/configure
@@ -16760,6 +16760,9 @@ fi
 
 if test "x$enable_symbol_versions" = xauto; then
 	case $host_os in
+		darwin*)
+			enable_symbol_versions=no
+			;;
 		# NOTE: Even if one omits -gnu on GNU/Linux (e.g.
 		# i486-slackware-linux), configure will (via config.sub)
 		# append -gnu (e.g. i486-slackware-linux-gnu), and this
--- a/configure.ac
+++ b/configure.ac
@@ -684,6 +684,9 @@ AC_ARG_ENABLE([symbol-versions], [AS_HELP_STRING([--enable-symbol-versions],
 	[], [enable_symbol_versions=auto])
 if test "x$enable_symbol_versions" = xauto; then
 	case $host_os in
+		darwin*)
+			enable_symbol_versions=no
+			;;
 		# NOTE: Even if one omits -gnu on GNU/Linux (e.g.
 		# i486-slackware-linux), configure will (via config.sub)
 		# append -gnu (e.g. i486-slackware-linux-gnu), and this
